cottontaildb_client/cottontaildb_cli.py

```diff
@@ -125,11 +130,45 @@ def entity(client, args):
     command = args.subcommand
     if command == 'drop':
         response = client.drop_entity(args.schema_name, args.entity_name)
-        print(response)
     elif command == 'about':
         response = client.get_entity_details(args.schema_name, args.entity_name)
-        print(response)
-
+    elif command == 'preview':
+        response = client.preview_entity(args.schema_name, args.entity_name, args.limit)
+    format(response)
```

Comment:
At this point the variable `response` might not be assigned yet if the command matches none of the above cases leading to unexpected behavior. Please either move the call to `format` to each branch of the if or check separately if the variable is assigned.